CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Added
 
 - [#2265](https://github.com/thanos-io/thanos/pull/2265) Compactor: Add `--wait-interval` to specify compaction wait interval between consecutive compact runs when `--wait` enabled.
+- [#2297](https://github.com/thanos-io/thanos/pull/2297) Store Gateway: Add `--experimental.enable-postings-compression` flag to enable reencoding and compressing postings before storing them into cache. Compressed postings take about 10% of the original size.
```

Comment:
This is not the only thing that changed. We changed overall the format of cached postings even with flag is set to false, right?

Reply:
Not really. If flag is set to false, then this store will only write original postings (as found in the index) to the cache. However, if store finds compressed entry in the cache, it will understand it and decompress.

Reply:
Fair (: